packages/manager/modules/pci/src/projects/project/storages/containers/container/delete/delete.controller.js

```diff
@@ -13,84 +13,27 @@ export default class PciBlockStorageDetailsDeleteController {
   }
 
   $onInit() {
-    this.loadings = {
-      init: false,
-      save: false,
-    };
-
-    this.initLoaders();
-  }
-
-  initLoaders() {
-    this.loadings.init = true;
-    return this.$translate.refresh()
-      .then(() => this.loadMessages())
-      .then(() => this.PciProjectStorageContainersService
-        .getContainer(this.projectId, this.containerId))
-      .then((container) => {
-        this.container = container;
-        return this.container;
-      })
-      .catch((err) => {
-        this.CucCloudMessage.error(
-          this.$translate.instant(
-            'pci_projects_project_storages_containers_container_delete_error_load',
-            { message: get(err, 'data.message', '') },
-          ),
-          'pci.projects.project.storages.containers.delete',
-        );
-      })
-      .finally(() => {
-        this.loadings.init = false;
-      });
-  }
-
-  loadMessages() {
-    return new Promise((resolve) => {
-      this.CucCloudMessage.unSubscribe('pci.projects.project.storages.containers.delete');
-      this.messageHandler = this.CucCloudMessage.subscribe(
-        'pci.projects.project.storages.containers.delete',
-        {
-          onMessage: () => this.refreshMessages(),
-        },
-      );
-      resolve();
-    });
-  }
-
-  refreshMessages() {
-    this.messages = this.messageHandler.getMessages();
+    this.isLoading = false;
   }
 
   deleteStorage() {
-    this.loadings.save = true;
+    this.isLoading = true;
     return this.PciProjectStorageContainersService
       .deleteContainer(this.projectId, this.container)
-      .then(() => {
-        this.CucCloudMessage.success(
-          this.$translate.instant(
-            'pci_projects_project_storages_containers_container_delete_success_message',
-            {
-              container: this.container.name,
-            },
-          ),
-          'pci.projects.project.storages.containers',
-        );
-        return this.goBack(true);
-      })
-      .catch((err) => {
-        this.CucCloudMessage.error(
-          this.$translate.instant(
-            'pci_projects_project_storages_containers_container_delete_error_delete',
-            {
-              message: get(err, 'data.message', null),
-            },
-          ),
-          'pci.projects.project.storages.containers.delete',
-        );
-      })
+      .then(() => this.goBack(this.$translate.instant(
```

Comment:
Sending a message to a navigation function looks weird to me

I understand what it does but It feels like `goBack` does too much, it should go back to the previous state/page, not naviguate and display a message